### PR TITLE
GIP-0066 and GIP-0068

### PR DESCRIPTION
**Pull request by tmigone**

### CODE REVIEW

File: gips/0066-graph-horizon.md
Comment (PedroMD) [inline gips/0066-graph-horizon.md:25]: > The Graph launched over two years ago

If we're referring to the network (mainnet), shouldn't we say 4 years, or reference the actual date that is December 2020?

File: gips/0066-graph-horizon.md
Comment (PedroMD) [inline gips/0066-graph-horizon.md:107]: >****service provider 

markdown formatting error?

File: gips/0066-graph-horizon.md
Comment (PedroMD) [inline gips/0066-graph-horizon.md:113]: > (for example, Graphtronauts, Graphscan, Indexer Tools, etc).

minor comment: GraphSeer could also be listed here

File: gips/0066-graph-horizon.md
Comment (PedroMD) [inline gips/0066-graph-horizon.md:107]: > In general, the more stake the service provider has on a provision the safer delegators will be.

Is this true, considering the slashable amount is defined on a data service basis, as mentioned in the section below? Or is it generally true, considering that some rules are _enforced/(encouraged?)_ by the Data Service Framework?

Comment (PedroMD) [reply gips/0066-graph-horizon.md:107]: The comment above didn't take into account this part found below.

> As a general guideline however, the Data Service framework proposes the slashing amount should be proportional to the provisioned stake at the time a dispute is raised.

So I guess this is generally true if the Data Service follows the Frameworks' guidelines.

File: gips/0066-graph-horizon.md
Comment (PedroMD) [inline gips/0066-graph-horizon.md:66]: These diagrams were exported without a background, which makes them hard to read if rendered in dark backgrounds (like here on Github if using the dark theme). Maybe consider updating with a white background like some other diagrams found in other sections of this GIP?

**Review by pcarranzav — APPROVED** [review]
These have been published now so I suggest merging and addressing @p-diogo's feedback in new PRs

**Comment by RembrandtK** [comment]
I see approved, but not closed, and not merged onto main. We are ready to merge and close this PR?

**Comment by tmigone** [comment]
Yes @RembrandtK, I'll merge now.

**Comment by tmigone** [comment]
Actually I can't merge as I don't have write access, @p-diogo can you help here?

**Pull request merged**
